frontend/kick-covid-19/src/pages/Blog.js

```diff
@@ -22,8 +23,11 @@ class Blog extends React.Component {
   render() {
     return (
       <div className="row">
-        {this.state.posts.map((body, id) => (
-          <Post body={body} key={id} />
+        {this.state.posts.map((body, posts, id) => (
+          <Link key={posts.id} to={`/posts/${posts.id}`}>
+            <Post body={body} key={id} />
+            <pre>{JSON.stringify(posts.body, null, 2)}</pre>
+          </Link>
```

Comment:
and then once you fix the restructuring issue this will need to be
```suggestion
          <Link key={_id} to={`/posts/${_id}`}>
            <Post body={body} />
            <pre>{JSON.stringify(posts.body, null, 2)}</pre>
          </Link>
```

Reply:
you don't need a key on the `Post` component just the parent `Link` and you wouldn't want to use the same key for them since they ought to be unique on the page